### ISSUE TITLE
Tabbing skips many things

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#489]
Tabbing through the results, the bookmark button is skipped and so is the title and the author link, access online line and paging

**Comment by cdmo** [comment]
Think we should see if Michelle can test this for us and see specifically how it's failing, but seems rather problematic at the moment

**Comment by binkylush** [comment]
Reviewed with Michelle McManus - all accessibility issues corrected. Things looked good and she was able to tab through all buttons and navigation without anything being skipped!
